hikari/api/rest.py

```diff
@@ -6849,14 +6851,18 @@ async def create_slash_command(
             a global command rather than a guild specific one.
         options : hikari.undefined.UndefinedOr[typing.Sequence[hikari.commands.CommandOption]]
             A sequence of up to 10 options for this command.
+        name_localizations : hikari.undefined.UndefinedOr[typing.Mapping[str, str]]
+            The name localizations for this command.
+        description_localizations : hikari.undefined.UndefinedOr[typing.Mapping[str, str]]
+            The name localizations for this command.
         default_member_permissions : typing.Union[hikari.undefined.UndefinedType, int, hikari.permissions.Permissions]
             Member permissions necessary to utilize this command by default.
 
             If `0`, then it will be available for all members. Note that this doesn't affect
             administrators of the guild and overwrites.
         dm_enabled : hikari.undefined.UndefinedOr[builtins.bool]
             Whether this command is enabled in DMs with the bot.
-
+            
```

Comment:
Trailing whitespace. Only going to comment on this one, you can fix it by doing `nox -s reformat-code`